### ISSUE TITLE
[SampleCard-FlightDetails.json; Config-Cortana-skills-classic]: The luminosity ratio for the text "Flight to JFK" is 3.67:1 and should be atleast 4.5:1

### ISSUE THREAD

**Issue by Bhaskar301101** [microsoft/AdaptiveCards#8478]
### Target Platforms

Android

### SDK Version

 2023.03.31.1

### Application Name

Adaptive Card

### Problem Description

Test Environment:
OS: Android 12
Device: S21+
Screen reader: Talkback
AdaptiveCard Android App version:  2023.03.31.1

Pre-requisite:
Import-Sample Cards->FlightDetails.json; Config-Cortana-skills-classic

Repro Steps:

1. Navigate to the Visual.
2. Navigate to the sample card.
3. Check color contrast ratio for the text "Flight to JFK"
4. Observe the issue.

Actual Result:
The luminosity ratio for the text "Flight to JFK" is 3.67:1 and should be atleast 4.5:1

Expected Result:
The luminosity ratio for the text "Flight to JFK" is 3.67:1 and should be atleast 4.5:1

Note:
This issue is also repro on below configs for text "Flight to JFK"
Cortana-skills-light.json (CCA is 4:1)
Outlook-desktop.json (CCA is 3.21:1)
Rener-to-Image.json (CCA is 3.21:1)
Skype.json (CCA is 2.68:!)
Windows-Notification.json (CCA is 3.56:1)
Windows-Timeline.json (CCA is 2.88:1)

User Experience:
Low vision users will might not able properly tell the text if color contrast is not sufficient.

WCAG Reference:
https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “,

### Screenshots

![MAS1 4 3SampleCard-FlightDetails json; Config-Cortana-skills-classic](https://user-images.githubusercontent.com/89897257/232746390-34cbbb18-3f48-4796-9a4e-5a2c31fdc69a.jpg)

### Card JSON

```JSON
NA
```

**Comment by jwoo-msft** [comment]
duplicate issue of hostconfig configuration options.
